cmd/cephcsi.go

```diff
@@ -52,6 +55,11 @@ var (
 	// cephfs related flags
 	volumeMounter = flag.String("volumemounter", "", "default volume mounter (possible options are 'kernel', 'fuse')")
 	mountCacheDir = flag.String("mountcachedir", "", "mount info cache save dir")
+
+	// livenes related flags
+	port             = flag.String("port", "8080", "TCP port for listening requests")
```

Comment:
need to update port and livenessendpoint description, can it be more descriptive?

can we change `seconds for between liveness polls `  to   `time interval in seconds between each poll` ?